build.gradle

```diff
@@ -11,6 +11,14 @@ buildscript {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
         classpath "com.synopsys.integration:common-gradle-plugin:${managedCgpVersion}"
     }
+
```

Comment:
please add a a comment that this will be removed in the future when spring is updated.